plugins/infinispan52/src/main/java/org/radargun/service/InfinispanServerLifecycle.java

```diff
@@ -23,11 +23,11 @@ protected Pattern getStartOK() {
    }
 
    protected Pattern getStartError() {
-      return Pattern.compile(".*\\[org\\.jboss\\.as\\].*started in.*");
+      return Pattern.compile(".*\\[org\\.jboss\\.as\\].*stopped in.*");
```

Comment:
Hi @diegolovison , I think the message here should be not `stopped` but:
Pattern.compile(".*\\[org\\.jboss\\.as\\].*started \\(with errors\\) in.*");
(at least this is how it was before Gustavo's changes)
Would be great if you will fix that too. Then I will rebase the PR as I am facing this issue too.